packages/mdc-textfield/_mixins.import.scss

```diff
@@ -14,6 +14,6 @@
 @forward "@material/ripple" as mdc-ripple-* hide $mdc-ripple-states-wash-duration, mdc-ripple-states-base-color, mdc-ripple-states-opacities, mdc-ripple-states-hover-opacity, mdc-ripple-states-focus-opacity, mdc-ripple-states-focus-opacity-properties-, mdc-ripple-states-press-opacity, mdc-ripple-states, mdc-ripple-states-activated, mdc-ripple-states-selected, mdc-ripple-states-interactions-, mdc-ripple-states-opacity, mdc-ripple-states-opacities-;
 @forward "./icon" as mdc-text-field-*;
 @forward "./index" as mdc-text-field-*;
-@forward "./helper-text" as mdc-text-field-*;
-@forward "./character-counter" as mdc-text-field-*;
+@forward "./helper_text" as mdc-text-field-*;
```

Comment:
@patrickrodee I'm not sure why this is changing with copybara, just revert it on the branch?

Reply:
It's likely a result of not being synced to head on google3. If you sync to head then re-export your CL, these will likely go away.

Reply:
Reverting for now, will investigate with @abhiomkar tomorrow.

Reply:
cl/293009144 will fix this.